handbook/engineering/environments.md

```diff
@@ -0,0 +1,32 @@
+# Environment
+
+This page describes our different cloud environments.
+
+## Google Cloud
+
+We utilize multiple Google Cloud projects and folders to organize our workloads and manage access control.
```

Comment:
Does someone need to request access or does everyone have access to these projects? 
If you need to request access, it might be helpful to have a note here on where to ask.

Reply:
Right now, it has to be requested, once we have it sorted out, I would like to for all engineers to have at least view access to most the projects.